### DISCUSSION TITLE
How can I adjust the speed of emulation?

### DISCUSSION THREAD

**Discussion by phirestalker** [giongto35/cloud-game#400] [Q&A]
When running the included Super Mario Bros game, the timer is counting down too fast and the music seems to be sped up. I see in the config file that I can set options for the emulation core. What options would I need to use to make the speed more realistic?

**Comment by sergystepanov** [comment]
I think, most of the Mario ROMs have emulation speed problems. There either NTSC 60Hz system running a PAL 50Hz game, or just how this ROM is running.
If you want slowed NTSC Mario, find World version on crocdb.
You can also force core region to PAL in the options. See how to set them in the config.yaml, and the actual options list is in the Libretro core files: https://github.com/libretro/nestopia/blob/master/libretro/libretro_core_options.h.
